lib/migrations/createGenesisDao.ts

```diff
@@ -1,10 +1,16 @@
-import { promisify } from "es6-promisify";
 import { Web3 } from "web3";
```

Comment:
How about moving this script out of the **migration** folder to something like **scripts** folder ?

Reply:
Done